schemas/oic.sec.roles-Update.json

```diff
@@ -4,11 +4,10 @@
   "title": "Device Roles information",
   "definitions": {
     "oic.sec.cred": {
-      "type": "object",
-      "properties": {
```

Comment:
I'm not 100% sure but I think our tooling required the "properties" tag so removing it could cause some problems.

Reply:
Nope ... the tooling will pull in whatever is at the JSON pointer. This is fine.